doc/source/development/debugging_extensions.rst

```diff
@@ -0,0 +1,37 @@
+.. _debugging_c_extensions:
+
+{{ header }}
+
+**********************
+Debugging C extensions
+**********************
+
+Pandas uses select C extensions for high performance IO operations. In case you need to debug segfaults or general issues with those extensions, the following steps may be helpful. These steps are geared towards using lldb as a debugger, though the steps for gdb will be similar.
+
+First, be sure to compile the extensions with the appropriate flags to generate debug symbols and remove optimizations. This can be achieved as follows:
+
+.. code-block:: sh
+
+   python setup.py build_ext --inplace -j4 --with-debugging-symbols
+
+Next you can create a script that hits the extension module you are looking to debug and place it in the project root. Thereafter launch a Python process under lldb:
+
+.. code-block:: sh
+
+   lldb run python
```

Comment:
looks like gdb doesnt' want the "run"?

Reply:
Cool thanks - it probably makes sense to just list both options explicitly